rnd/autogpt_server/autogpt_server/executor/manager.py

```diff
@@ -343,6 +361,8 @@ def on_node_execution(cls, q: ExecutionQueue[NodeExecution], data: NodeExecution
 
     @classmethod
     def on_graph_executor_start(cls):
+        cls.loop = asyncio.new_event_loop()
+        cls.loop.run_until_complete(db.connect())
```

Comment:
@Pwuts, I know that you've been working on a PR to properly clean this, yet I'm adding a new one.
Just want to align on whether I should pull and reconcile your PR or vice-versa.